portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -39,4 +36,30 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     response.setContentType("application/json;");
     response.getWriter().println(json);
   }
-}
+
+@Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Get the input from the form.
+    String text = getParameter(request, "text-input", "");
+    messagesList.add(text);
+
+    // Respond with the result.
+    response.setContentType("text/html;");
+    response.getWriter().println(Arrays.asList(messagesList));
```

Comment:
no need to print if you're redirecting (since the redirect is the only thing htat will show)

Reply:
Ohh that makes more sense! Changed! thank you